projects/admin/src/app/deposit/editor/editor.component.html

```diff
@@ -28,17 +28,46 @@
 <ng-container *ngIf="deposit">
   <form [formGroup]="form" (ngSubmit)="save()">
     <div class="row mb-5">
-      <div class="col-sm-8"></div>
-      <div class="col-sm-4 text-left text-sm-right">
-        <div class="row">
-          <div class="col">
-            <button type="submit" class="btn btn-primary btn-block btn-lg">
+      <div class="col-sm-8">
+        <div class="row" *ngIf="view === 'form'">
+          <div class="col-12 col-sm-6">
+            <div class="btn-group" dropdown *ngIf="currentStep === 'metadata'">
+              <button
+                id="import-button"
+                dropdownToggle
+                type="button"
+                class="btn btn-primary dropdown-toggle"
+                aria-controls="dropdown-import"
+              >
+                <i class="fa fa-mail-forward mr-2"></i> {{ 'Import metadata' | translate }}
+                <span class="caret"></span>
+              </button>
+              <ul
+                id="dropdown-import"
+                *dropdownMenu
+                class="dropdown-menu"
+                role="menu"
+                aria-labelledby="button-basic"
+              >
+                <li role="menuitem">
+                  <a class="dropdown-item" href (click)="extractPdfMetadata($event)">{{
```

Comment:
put the translate in the <a>

Reply:
We take this as a standard ? The directive instead of pipe ? If yes, I will do another PR to correct all templates.